### PR TITLE
242 we should also have a permission screen that request permission to collect crash reports to match gdpr compliance

### LINKED ISSUE #242: We should also have a permission screen that request permission to collect crash reports. To match gdpr compliance
We should also have a permission screen that request permission to collect crash reports. To match gdpr compliance

_Originally posted by @vanlooverenkoen in https://github.com/icapps/flutter-template/issues/241#issuecomment-1090538688_

### PR DESCRIPTION
**Pull request by rwillemsandroid**
fixes #242

### CODE REVIEW

**Review by NicolaVerbeeck — CHANGES_REQUESTED** [review]
Same help file, no need to duplicate.
Also add screenshot tests for phone and tablet

**Comment by vanlooverenkoen** [comment]
Fix some merge conflicts

File: test/widget/debug/select_language_dialog_test.dart [hunk 1]
```diff
@@ -19,16 +20,16 @@ void main() {
...
-    when(globalViewModel.isLanguageSelected('nl')).thenAnswer((_) => false);
-    when(globalViewModel.isLanguageSelected(null)).thenAnswer((_) => false);
-    final widget = SelectLanguageDialog(goBack: () {});
-
-    await TestUtil.loadScreen(tester, widget);
-    await TestUtil.takeScreenshot(tester, 'select_language_dialog_initial_state');
-
-    verify(globalViewModel.isLanguageSelected(any));
-    verifyGlobalViewModel();
+    // when(globalViewModel.isLanguageSelected('en')).thenAnswer((_) => true);
```
Comment (NicolaVerbeeck) [inline test/widget/debug/select_language_dialog_test.dart:23]: Test gone?

**Review by NicolaVerbeeck — APPROVED** [review]

File: test/widget/debug/select_language_dialog_test.dart (hunk 1 above)
Comment (rwillemsandroid) [reply test/widget/debug/select_language_dialog_test.dart]: hmm, no sure how that happened, restored 👍

**Pull request merged**